### PR TITLE
Add common linters as test dependencies

### PR DESCRIPTION
**Pull request by Blast545**
Included ament_lint_common and ament_lint_auto as test dependencies

Signed-off-by: Jorge J. Perez <jjperez@ekumenlabs.com>

### CODE REVIEW

**Comment by dirk-thomas** [comment]
Since this is an empty package what benefit do the added tests provide?

**Comment by Blast545** [comment]
> Since this is an empty package what benefit do the added tests provide?

@dirk-thomas This is part of the initial efforts to get all ROS-Core packages to quality level 1, as defined by [REP2004](https://github.com/ros-infrastructure/rep/blob/rep-2004/rep-2004.rst) 

This addition ensures XML linters are run for the package.xml file

File: common_interfaces/package.xml [hunk 1]
```diff
@@ -9,6 +9,9 @@
 
   <buildtool_depend>ament_cmake</buildtool_depend>
 
+  <test_depend>ament_lint_common</test_depend>
+  <test_depend>ament_lint_auto</test_depend>
+
```
Comment (dirk-thomas) [inline common_interfaces/package.xml]: Nitpick: test dependencies should be listed below `exec_depend`s.

File: common_interfaces/package.xml [hunk 2]
```diff
@@ -22,6 +22,9 @@
   <exec_depend>trajectory_msgs</exec_depend>
   <exec_depend>visualization_msgs</exec_depend>
 
+  <test_depend>ament_lint_common</test_depend>
+  <test_depend>ament_lint_auto</test_depend>
```
Comment (dirk-thomas) [inline common_interfaces/package.xml]: Nitpick: in alpha order.

**Comment by Blast545** [comment]
Linters pass locally, and this package is blacklisted on CI so it can't be tested there at the moment. I'll proceed to merge.

**Pull request merged**